docs/index.html

```diff
@@ -288,20 +288,54 @@ <h2>#TwitterThrowback</h2>
     var submitButton = document.getElementById('submit-button');
     var loadingIndicator = document.getElementById('loading-indicator');
 </script>
+<script>
+    /**
+     * @function getMonthAndDay
+     * @description returns the Date API Friendly month and day 
+     * @param monthAndDayString a string containing the month and day separated by a space
+    */
+     function getMonthAndDay(monthAndDayString) {
+        const todayAsArray = [new Date().getMonth() + 1, new Date().getDate()];
+        if (monthAndDayString.split(' ').length < 2) {
+            monthAndDayString = monthAndDayString.concat(` ${todayAsArray[1]}`);
+        }
+        const dummyDate = new Date(monthAndDayString);
```

Comment:
Neat trick. Handles validation/normalization for us.👍

Reply:
Yh, that gave me headaches.